contracts.go

```diff
@@ -11,25 +11,53 @@ import (
 	"regexp"
 	"strings"
 	"text/template"
-
-	"github.com/spf13/cobra"
 )
 
 var (
-	EXHIBIT_PREFIX = "exhibit."
-	SIGN_PREFIX    = "sign."
-	VALUE_SUFFIX   = ".value"
+	VAR_PREFIX     = ".Var."
+	EXHIBIT_PREFIX = ".Exhibit."
+	SIGN_PREFIX    = ".Sign."
+	// [zr] not sure what to do with suffix ...
+	VALUE_SUFFIX = ".value"
 
 	VARIABLE_REGEX = regexp.MustCompile(`{{\s*([\w\._-]+)\s*}}`)
 )
 
+// for reading a contract template
 type ContractTemplate struct {
 	TemplateHash string
 	Vars         []string
 	Exhibits     []string
 	Signing      []string
 }
 
+type Var struct {
```

Comment:
I don't think we can do this. Maybe there is some set of common vars that every contract will have, but in general we cant get all the vars in a structure because they can be specific to the contract. Same with exhibits - it makes no sense to have a struct for them